lib/src/widgets/help_widget/help_widget.dart

```diff
@@ -49,12 +49,14 @@ class HelpWidget extends StatelessWidget {
 
     return Container(
       constraints: constraints,
-      child: ValueListenableBuilder<HelpWidgetView>(
-        valueListenable: _controller,
-        builder: (context, view, _) {
+      child: AnimatedBuilder(
+        animation: _controller,
+        builder: (context, _) {
+          final view = _controller.currentItem;
+
```

Comment:
Not really sure about the switch from ValueNotifier to ChangeNotifier, and ValueListenableBuilder -> AnimatedBuilder.

What's your reasoning here?

Here's my thinking:

Per docs, ValueNotifier is `/// A [ChangeNotifier] that holds a single value.`.
In our use case, the single value is `view`, and `didTraverseForward` is just a detail on how the transition between values is to be made.

`AnimatedBuilder` is off-putting, since we already have a `HelpWidgetAnimationBuilder` below. It makes me think that there's some nested animation here.

Reply:
And that said, I'd rather keep using ValueNotifier+ValueListenableBuilder combo.

Reply:
But we don't use the [ValueNotifier.value] (only in constructor), and call [notifyListeners] ourselves.

I agree about naming though, we might want to rename [HelpWidgetAnimationBuilder]?
Or we can merge HelpWidgetAnimation into HelpWidgetAnimationBuild?

Reply:
On `HelpWidgetAnimationBuilder` -- no, I'd rather keep that piece as-is. It does cleanly encapsulate a well-defined piece of functionality. Could probably use some refactoring though, but let's leave it for later.

Might rename it though -- something like `ViewTransitionBuilder`. Then it makes sense -- we have a widget that determines which view to show (`ValueListenableBuilder`/`AnimatedBuilder`?), and a widget which determines how to transition to it.

Reply:
As for `AnimatedBuilder` -- looks like we're OK to keep it. There seems to have been some discussion on the matter already: https://github.com/flutter/flutter/issues/95967.